insonmnia/miner/cgroup_nonlinux.go

```diff
@@ -0,0 +1,19 @@
+// +build !linux
+
+package miner
+
+import (
+	"fmt"
+	"runtime"
+)
+
+type Resources interface{}
+
+const (
+	platformSupportCGroups = false
+	parentCgroup           = ""
+)
+
+func initializeControlGroup(*Resources) (cGroupDeleter, error) {
+	return nil, fmt.Errorf("%s does not support Control Groups", runtime.GOOS)
```

Comment:
How about some kind of NilDeleter instead of error? Looks like it is not very usefull. And we already have a flag indicating cgroups support